packages/pkgpanda-api/build

```diff
@@ -12,4 +12,9 @@ cp /pkg/extra/dcos-pkgpanda-api.socket $systemd_socket
 
 systemd_service=$PKG_PATH/dcos.target.wants/dcos-pkgpanda-api.service
 mkdir -p $(dirname $systemd_service)
-cp /pkg/extra/dcos-pkgpanda-api.service $systemd_service
+envsubst '$PKG_PATH' < /pkg/extra/dcos-pkgpanda-api.service > "$systemd_service"
```

Comment:
I believe `envsubst '${PKG_PATH}' ...` is more explicit per the [SHELL-FORMAT docs](https://www.gnu.org/software/gettext/manual/gettext.html#sh_002dformat)